dvc.yaml

```diff
@@ -0,0 +1,19 @@
+stages:
+  get_data:
+    cmd: python src/get_data.py
+    deps:
+    - src/get_data.py
+    outs:
+    - data
+  preprocessing:
+    cmd: python src/preprocessing.py
+    deps:
+    - src/preprocessing.py
```

Comment:
The dependencies are not correct. This should depend on the files that created by the get_data step. Otherwise, preprocessing won't run again when we change data